server/config.ts

```diff
@@ -33,6 +33,7 @@ export interface ApiConfig {
 
 export default {
   https: production,
+  applicationName: 'Recall Decisions',
```

Comment:
Might want to add the work "Make" to the application name

Reply:
I asked Paul this yesterday and he definitely confirmed it's 'Recall Decisions', as in the prototype header

Reply:
At least for the time being 🤷